### ISSUE TITLE
Remember more than one day?

### ISSUE THREAD

**Issue by arnauldb** [tecoholic/Just-Arrived#2]
Hello,
Does it remember tweets more than one day? What if I see a tweet I already saw 3 days ago? Will it be dimmed too?
Thank you for this addon.

**Comment by tecoholic** [comment]
@arnauldb No. It doesn't. It remembers only the tweets that are loaded into the browser on that day.

**Comment by arnauldb** [comment]
It could be a nice addition to be able to select the number of remembered days and have a dim option (to dim more or less). It is a very useful addon, I am using it now, thks.
